### DISCUSSION TITLE
Change location of job dlls

### DISCUSSION THREAD

**Discussion by colby02184** [quartznet/quartznet#479] [General] [answered]
Does Quatrz have a way to change where job dlls are stored? I have quartz running in a Windows service and I'd like to keep the directory of the service as clean as possible.

**Comment by lahma** [comment, marked as answer]
This is not directly related to Quartz, there's [discussion on SO](http://stackoverflow.com/questions/1373100/how-to-add-folder-to-assembly-search-path-at-runtime-in-net) about how to add searched assembly locations. You can also provide custom ITypeLoadHelper for Quartz to use for loading types.
